### PR TITLE
refactor: add theme toggle

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/861mrtzmx

TODO:

- [x] Remove old toggle
- [x] Update ui dep (https://github.com/ArkEcosystem/laravel-foundation/pull/559)

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: tests/Unit/Services/Blockchain/NetworkTest.php [hunk 1]
```diff
@@ -18,6 +18,9 @@
     expect($subject->alias())->toBe($config['alias']);
     expect($subject->api())->toBe($config['api']);
     expect($subject->explorerTitle())->toBe(config('app.name'));
+    expect($subject->mainnetExplorerUrl())->toBe($config['mainnetExplorerUrl']);
+    expect($subject->testnetExplorerUrl())->toBe($config['testnetExplorerUrl']);
+    expect($subject->polygonExplorerUrl())->toBe($config['polygonExplorerUrl']);
```
Comment (ItsANameToo) [inline tests/Unit/Services/Blockchain/NetworkTest.php:23]: we can get rid of this as well as it was only necessary to link migration transactions

Comment (ItsANameToo) [reply tests/Unit/Services/Blockchain/NetworkTest.php:23]: can be another PR

**Review by ItsANameToo — APPROVED** [review]

File: tests/Unit/Services/Blockchain/NetworkTest.php (hunk 1 above)
Comment (alexbarnsley) [reply tests/Unit/Services/Blockchain/NetworkTest.php:23]: :thinking: added this because it was failing tests

Comment (alexbarnsley) [reply tests/Unit/Services/Blockchain/NetworkTest.php:23]: Ah unless you mean the whole thing since you've merged :oleglala:

**Pull request merged**